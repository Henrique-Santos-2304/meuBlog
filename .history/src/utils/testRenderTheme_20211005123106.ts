import { ThemeProvider } from "styled-components";
import { render, RenderResult } from "@testing-library/react";
import theme from "../styles/theme";
export const renderTheme = (children: React.ReactNode): RenderResult => {
  return render(<ThemeProvider>{children}</ThemeProvider> );
};
