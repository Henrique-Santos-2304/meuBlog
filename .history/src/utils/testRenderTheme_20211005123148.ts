import { ThemeProvider } from "styled-components";
import { render, RenderResult } from "@testing-library/react";
import theme from "../styles/theme";
export const renderTheme = (children: React.ReactElement): RenderResult => {
  return render(<ThemeProvider theme={theme}>{children}</ThemeProvider> );
};
