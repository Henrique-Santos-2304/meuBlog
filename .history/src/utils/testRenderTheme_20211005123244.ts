import { ThemeProvider } from "styled-components";
import { render, RenderResult } from "@testing-library/react";
import theme from "../styles/theme";
export const renderTheme = (children) => {
  return render(<ThemeProvider theme={theme}>{children}</ThemeProvider> );
};
