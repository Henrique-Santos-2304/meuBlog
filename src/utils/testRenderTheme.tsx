import { ThemeProvider } from "styled-components";
import { ChakraProvider } from "@chakra-ui/react";

import { render, RenderResult } from "@testing-library/react";
import theme from "styles/theme";
export const renderTheme = (children: React.ReactNode): RenderResult => {
  return render(
    <ThemeProvider theme={theme}>
      <ChakraProvider> {children}</ChakraProvider>
    </ThemeProvider>
  );
};
