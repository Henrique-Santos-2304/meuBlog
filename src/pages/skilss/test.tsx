import "match-media-mock";
import Skilss from ".";
import { renderTheme } from "utils/testRenderTheme";

describe("<Skilss />", () => {
  it("should render ", () => {
    const { container } = renderTheme(<Skilss />);
    expect(container).toMatchSnapshot();
  });
});
