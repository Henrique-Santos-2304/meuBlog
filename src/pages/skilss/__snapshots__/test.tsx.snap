// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Skilss /> should render  1`] = `
.c0 {
  bottom: -1rem;
}

.c0 ul:last-child {
  color: orange;
}

.c0 ul li {
  color: orange;
}

.c0 ul li.slick-active button::before {
  color: orange;
}

.c0 ul li button::before {
  font-size: 1.4rem;
}

<div>
   
  <div
    class="chakra-container css-xawjhy"
  >
    <div
      class="css-k008qs"
    >
      <div
        class="chakra-container css-dw6p92"
      >
        <h2
          class="chakra-heading css-b8w7c"
        >
          Habilidades
        </h2>
        <div
          class="slick-slider slick-initialized"
          dir="ltr"
        >
          <button
            class="slick-arrow slick-prev slick-disabled"
            data-role="none"
            style="display: block;"
            type="button"
          >
             
            Previous
          </button>
          <div
            class="slick-list"
          >
            <div
              class="slick-track"
              style="opacity: 1; transform: translate3d(0px, 0px, 0px);"
            >
              <div
                aria-hidden="false"
                class="slick-slide slick-active slick-current"
                data-index="0"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Python"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416786/1024px-Python-logo-notext_jsr03y.png"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Python
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="1"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="JavaScript"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634417394/javascript-icon_zaobje.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      JavaScript
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="2"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Typescript"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416790/typescript_c5xdpy.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Typescript
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="3"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="HTML"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634417749/w3_html5-icon_sviivv.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      HTML
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="4"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="React"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416789/react_eugvgw.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      React
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="5"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Next.Js"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416788/nextjs_jbhm7d.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Next.Js
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="6"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Jest"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416787/jest_yvhqf3.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Jest
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="7"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Cypress"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416786/cypress_bmhnah.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Cypress
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="8"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Testing Library"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416789/rtl_i3scd8.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Testing Library
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="9"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Storybook"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416789/storybook_q1s5cl.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Storybook
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <button
            class="slick-arrow slick-next"
            data-role="none"
            style="display: block;"
            type="button"
          >
             
            Next
          </button>
          <div
            class="c0 slick-dots"
          >
            <ul>
               
              <li
                class="slick-active"
              >
                <button>
                  1
                </button>
              </li>
              <li
                class=""
              >
                <button>
                  2
                </button>
              </li>
              <li
                class=""
              >
                <button>
                  3
                </button>
              </li>
               
            </ul>
          </div>
        </div>
        <div
          class="slick-slider slick-initialized"
          dir="ltr"
        >
          <button
            class="slick-arrow slick-prev slick-disabled"
            data-role="none"
            style="display: block;"
            type="button"
          >
             
            Previous
          </button>
          <div
            class="slick-list"
          >
            <div
              class="slick-track"
              style="opacity: 1; transform: translate3d(0px, 0px, 0px);"
            >
              <div
                aria-hidden="false"
                class="slick-slide slick-active slick-current"
                data-index="0"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Styled Components"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634417550/styled-components-1_kdt29i.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Styled Components
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="1"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Bootstrap"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416786/Bootstrap_logo_utfaxr.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Bootstrap
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="2"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Chakra-Ui"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416786/54212428_dpejja.png"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Chakra-Ui
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="3"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Tailwind-Css"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634417414/tailwindcss-icon_1_xozsfd.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Tailwind-Css
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active"
                data-index="4"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Css"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634417704/w3_css-icon_ijitjf.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Css
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="5"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Strapi"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416790/strapi_e1wlna.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Strapi
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="6"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="Apolo"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416786/apollo_qhtdll.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      Apolo
                    </p>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="7"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    class="css-x5fiqn"
                  >
                    <img
                      alt="GraphQl"
                      class="chakra-image css-1xbzop4"
                      src="https://res.cloudinary.com/defnibbpl/image/upload/v1634416786/graphql_c3ix6o.svg"
                    />
                    <p
                      class="chakra-text css-18unoc9"
                    >
                      GraphQl
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <button
            class="slick-arrow slick-next"
            data-role="none"
            style="display: block;"
            type="button"
          >
             
            Next
          </button>
          <div
            class="c0 slick-dots"
          >
            <ul>
               
              <li
                class="slick-active"
              >
                <button>
                  1
                </button>
              </li>
              <li
                class=""
              >
                <button>
                  2
                </button>
              </li>
              <li
                class=""
              >
                <button>
                  3
                </button>
              </li>
               
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      class="chakra-container css-v4nxjc"
    />
  </div>
</div>
`;
